Services/DevicePropertiesRequest.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Shared;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Diagnostics;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Models;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.Services
+{
+    public interface IDevicePropertiesRequest
```

Comment:
the name is a bit strange, e.g. why "request" ? I would have chosen something like 
```
IDeviceProperties {
    Task OnChangeUpdateAsync(ISmartDictionary deviceProperties)
}
```